### PR TITLE
pbTests: Debian10 rsync folder changes

### PR DESCRIPTION
**Pull request by Willsparker**
Turns out the Debian 10 box I'm using, uses rsync folders and not the standard `vboxsf` that most boxes have. It's not a massive issue as we already support `FreeBSD12` that uses `rsync` folders, so it's just the addition of the condition.

### CODE REVIEW

**Comment by Willsparker** [comment]
Ref: latest commit
Forgot to add this condition to #1309

**Comment by Willsparker** [comment]
Ref: latest commit
Forgot to add this condition to #1309

File: ansible/Vagrantfile.Debian10 [hunk 1]
```diff
@@ -7,7 +7,7 @@ sudo apt-get install software-properties-common -y
 sudo apt-get install gpg -y	
 sudo apt-get update
 # Get IPs of the VM, and output to shared folder
-ip address show | grep -e "\\binet\\b" | cut -d' ' -f6 | cut -d/ -f1  >> /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp
+ip address show | grep -e "\\inet\\b" | cut -d' ' -f6 | cut -d/ -f1  >> /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook/hosts.tmp
```
Comment (sxa) [inline ansible/Vagrantfile.Debian10:10]: DId you mean to remove this `b`?

Comment (Willsparker) [reply ansible/Vagrantfile.Debian10:10]: Weirdly the give the same output - I'll change it back to keep it consistent with the Deb8 VF

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml [hunk 2]
```diff
@@ -116,6 +116,7 @@
   with_items: "{{ gcc_compiler }}"
   when:
     - (ansible_distribution_major_version != "9" and ansible_architecture != "armv7l")
+    - ansible_distribution_major_version != "10"
```
Comment (sxa) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml:119]: This doesn't seem related to the PR title.
Also shouldn't we be saying which distribution this is? When we hit RHEL/Cent10 this conditional will trigger

Comment (Willsparker) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Debian.yml:119]: Nope, this is within the `Common/tasks/Debian.yml` file ; which only runs on Debian guests. However, I am going to remove this because I've done it a better way in my own testing, and it relates to a PR I'll be doing later on today.
